be/src/exec/schema_scanner/schema_be_datacache_metrics_scanner.cpp

```diff
@@ -1,24 +1,45 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
 //
-// Created by Smith on 2023/11/30.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
 //
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 #include "exec/schema_scanner/schema_be_datacache_metrics_scanner.h"
 
-#include <agent/master_info.h>
-#include <block_cache/block_cache.h>
-#include <gutil/strings/substitute.h>
+#include "agent/master_info.h"
+#include "block_cache/block_cache.h"
+#include "column/datum.h"
+#include "gutil/strings/substitute.h"
+#include "runtime/string_value.h"
 
 namespace starrocks {
 
+TypeDescriptor SchemaBeDataCacheMetricsScanner::_dir_spaces_type = TypeDescriptor::create_array_type(
+        TypeDescriptor::create_struct_type({"path", "quota_bytes"}, {TypeDescriptor::from_logical_type(TYPE_VARCHAR),
+                                                                     TypeDescriptor::from_logical_type(TYPE_BIGINT)}));
+
+TypeDescriptor SchemaBeDataCacheMetricsScanner::_used_bytes_detail_type = TypeDescriptor::create_struct_type(
```

Comment:
Isn't map type better if you wanna add more priority in the future?

Reply:
Using Map<INT, BIGINT> instead.